core/domain/classifier_domain.py

```diff
@@ -113,10 +114,10 @@ def __init__(
                         'answers': ['a2', 'a3']
                     }
                 ]
-            classifier_data: dict. The actual classifier model used for
-                classification purpose.
-            data_schema_version: int. Schema version of the data used by the
-                classifier. This depends on the algorithm ID.
+            classifier_data: Object. The protobuf object containing classifier
```

Comment:
Same note re "Object", here and elsewhere.

Reply:
Done.